sorc/ncep_post.fd/AVIATION.f

```diff
@@ -208,7 +210,7 @@ SUBROUTINE CALICING (T1,RH,OMGA, ICING)
 !
       DO J=JSTA,JEND
         DO I=1,IM
-
+        IF(OMGA(I,J)<SPVAL.AND.T1(I,J)<SPVAL) THEN
          IF(OMGA(I,J) < 0.0 .AND.                       &
             (T1(I,J) <= 273.0 .AND. T1(I,J) >= 251.0)   &
               .AND. RH(I,J) >= 70.0) THEN
```

Comment:
@JesseMeng-NOAA I am wondering if adding RH(i,j)<SPVAL in line 213?
IF(OMGA(I,J)<SPVAL.AND.T1(I,J)<SPVAL.AND.RH(i,j)<SPVAL) THEN

Reply:
will do

Reply:
done